Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h

```diff
@@ -124,10 +124,10 @@ namespace OvEditor::Core
 		double m_lastMousePosX = 0.0;
 		double m_lastMousePosY = 0.0;
 		OvMaths::FVector3 m_ypr;
-		float m_mouseSensitivity = 0.05f;
-		float m_cameraDragSpeed = 0.01f;
+		float m_mouseSensitivity = 0.12f;
+		float m_cameraDragSpeed = 0.07f; 
```

Comment:
This one is a bit too much in my opinion. I've tested it and `0.03f` looked more appropriate. I know that this is totally subjective tho

Reply:
I had Unity open in order to compare the feeling. Maybe it was too much yes, but my general feeling was that the movement felt quite heavy

Reply:
I was actually only talking about the `m_cameraDragSpeed`, the `m_mouseSensitivity` of `0.12f` felt good IMO

Reply:
Okey! then `m_cameraDragSpeed` is set to `0.12` and `m_mouseSensitivity` to `0.03` I think is much better like this